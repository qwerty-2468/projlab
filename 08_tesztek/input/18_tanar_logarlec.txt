create room r
create teacher t
addperson t r
create sliderule sr
add sr r
pickup sr t

status sr