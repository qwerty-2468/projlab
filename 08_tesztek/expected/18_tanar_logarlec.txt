
sr
location: r
holder: NULL
